Esteban-Salas-Posada/tarea-1/style.css

```
@@ -0,0 +1,198 @@
+html{
+    font-size: 62.5%;
+    background-color: #b1b1b1;
+    margin: 0;
+    padding: 0;
+}
+img{
+    max-width: 100%;
+}
+h1{
+    text-transform: uppercase;
+    margin: 0;
+    padding: 0;
+}
+p{
+    text-transform: uppercase;
+    margin: 0;
+    padding: 0;
+}
+.main{
+    background-color: #BBDDF6;
+    background-image: url(../img/modelo1.png);
```

@esteban-salas , this path is not working for me please fix
```suggestion
    background-image: url(img/modelo1.png);
```